src/components/sidebar/Sidebar.jsx

```diff
@@ -41,4 +48,9 @@ const Sidebar = () => (
   </aside>
 );
 
+
+Sidebar.propTypes = {
+  userRoles: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Comment:
Why aren't we using some default userRoles instead of making them required. Seems to me like we should be able to have a sidebar that doesn't necessarily require userRoles. What do you think?